pages/_en-US/ds-index/hardmod.md

```diff
@@ -11,4 +11,90 @@ Hardmodding is when you physically solder the motherboard of the Nintendo DSi to
 ![Original DS motherboard with touch-points labeled](https://media.discordapp.net/attachments/547986366357700620/736370094392999947/mobo_pinout.png)
 
 ### Nintendo DSi
-Follow the "Hardware NAND Mod Guide" and "Decrypting your NAND" of [this](https://gbatemp.net/threads/dsi-downgrading-the-complete-guide.393682/) guide.
+
+To hardmod a Nintendo DSi / DSi XL (LL) you will need:
+  - [Pencil tipped soldering iron](https://www.amazon.com/Sywon-Soldering-Station-Temperature-Adjustable/dp/B01N4571Q6/)
+  - [Very fine wire](https://www.amazon.com/BNTECHGO-Silicone-Flexible-Stranded-Impedance/dp/B01MXGNTA4/ref=sr_1_8?crid=1Y1M31ZLILOYN&dchild=1&keywords=30+awg+wire&qid=1608818117&sprefix=30+AWG+%2Caps%2C250&sr=8-8), 28AWG or smaller, preferably 30AWG+ 
+  - SD to microSD adapter
+  - An SD card reader able to read eMMC chips running in single data-line mode. [Here's a known working one](https://www.amazon.com/gp/product/B006T9B6R2)
+  - [Win32DiskImager](https://sourceforge.net/projects/win32diskimager/) and [HxD](https://mh-nexus.de/en/downloads.php?product=HxD20) if on Windows
+  - A valid DSi backup of the DSi you are hardmodding
+  - To be comfortable soldering to pads that are as small as 0.5mm in diameter
+
+#### DSi side A pinouts
+![DSi side A pinouts](/assets/images/dsi-hardmod/side_a.jpg)
+#### DSi side B pinouts
+![DSi side B pinouts](/assets/images/dsi-hardmod/side_b.jpg)
+#### DSi XL side B pinouts
+![DSi XL side B pinouts](/assets/images/dsi-hardmod/dsi_xl_side_b.png)
+
+- You will need to solder the points on the motherboard to the microSD card adapter
+  - CMD to pin 2
+  - GND to pins 3 and 6
+  - CLK to pin 5
+  - DAT0 to pin 7
+
+#### Soldered microSD adapter example
+![microSD example](/assets/images/dsi-harmod/sd.jpg)
+
+- Insert the microSD card adapter to your computer
+  - **WARNING** - Windows will ask you to format it: **DO NOT** format it - this might cause irrepairable damage
+
+#### Windows instructions (skip for macOS / Linux)
+
+1. Open Win32DiskImager
+2. Click the folder icon and browse to your desktop. In the text box, write NAND_0.bin. When selecting type, choose All types *.* 
+3. Choose the device that is the DSi and click read. 
+4. Once it is done, click the folder icon,change the name to NAND_1.bin and read again
+5. Open HxD and drag both files into the editor. Go to the top bar, click "Analysis",click "File compare" from the drop down menu, then click "Compare".
+6. Choose to compare both files then click OK when done
+  - If it says "The chosen files are identical.", you are good to go to the next section
+  - If it doesn't say that, and both NANDs aren't around 240MB, dump NAND_1/NAND_0 again
+7. Open Win32DiskImager, click the folder icon and choose your existing NAND backup you created before.
+8. Flash it with the `Write` button.
+9. Unplug the SD card adapter and attempt to turn on the DSi
```

Comment:
Minor thing, but I usually do markdown lists like
```md
1. thing
1. another thing
1. yet more things
```
as it auto does the numbers
1. thing
1. another thing
1. yet more things